mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -831,6 +849,10 @@ The new line here must be kept as the second half of the string is clickable for
     <!-- Preference for using following device theme -->
     <string name="preference_follow_device_theme">Follow device theme</string>
 
+    <!-- The title of the error page for websites that do not support HTTPS when HTTPS-Only is turned on  -->
+    <string name="errorpage_httpsonly_title">The site you requested does not support HTTPS</string>
+    <!-- The text of the error page for websites that do not support HTTPS when HTTPS-Only is turned on. -->
```

Comment:
@pocmo Please add in comments what the placeholder is replaced with

Reply:
The change finally made it over.